pkg/k8s/client.go

```diff
@@ -31,6 +32,9 @@ func (c *Client) RecreatePod(pod *apiv1.Pod) error {
 	if err := c.DeletePod(pod); err != nil {
 		return err
 	}
+	// TODO: recreation in seperate go routine
+	time.Sleep(10 * time.Second)
```

Comment:
that's not going to work when the pod has a long terminationgrace ... need to loop until the pod is gone and timeout in case something else re-created it (for example if it was a static pod) ... so better keep the resourceId around and when it comes back with a different id then it's good ... also this won't work for static pods at all, so need a workaround for that too ... maybe best to just not do this at all and leave it  as an open issue with all the gotschas documented instead of adding a buggy solution that we don't need atm